frontend/src/components/profile/FeaturedProfileCard.tsx

```diff
@@ -0,0 +1,73 @@
+import {
```

Comment:
I don't see where this would be used tbh - I see it's the same as the favourite events/communities but we're not going to be highlighting users anywhere afaik.